solutions/회의실_배정.js

```diff
@@ -0,0 +1,30 @@
+const INPUT_PATH = "./inputs/회의실_배정.txt";
+
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : INPUT_PATH;
+const input = fs.readFileSync(filePath).toString().trim().split("\n");
+
+function solution(inputArguments) {
+  const [N, ...list] = inputArguments;
+  const meetingTimes = [];
+  let currentTime = 0;
+  let meetingCount = 0;
+
+  for (let i = 1; i <= N; i++) {
+      const [startTime, endTime] = list[i].split(" ").map(Number);
+      meetingTimes.push({ startTime, endTime });
+  }
```

Comment:
```suggestion
const meetingTimes = list.map((item) => {
        const [startTime, endTime] = item.split(" ").map(Number);
        
        return  ({ startTime, endTime });
});
```
위와 같이 map 메서드를 사용할 수도 있을거 같습니다